common/app/views/fragments/newHeader.scala.html

```diff
@@ -82,7 +85,7 @@
                        tabindex="0"
                        data-link-name="nav2 : veggie-burger : show">
                     <span class="veggie-burger__icon"></span>
-                    <span class="veggie-burger__label">Menu</span>
+                    <span class="veggie-burger__label @if(page.metadata.hasSlimHeader) {u-h}">Menu</span>
```

Comment:
`@RenderClasses` for better readability? There will be another case, where the subnavigation is hidden :)